src/google-classroom-extractor/edfi_google_classroom_extractor/api/courses.py

```diff
@@ -20,6 +20,178 @@
 logger = logging.getLogger(__name__)
 
 
+def request_course_aliases(
+    resource: Optional[Resource], course_id: str
+) -> List[Dict[str, str]]:
+    """
+    Fetch CourseAlias API data for the given course id
+
+    Parameters
+    ----------
+    resource: Optional[Resource]
+        a Google Classroom SDK Resource
+    course_id: str
+        a Google Classroom course id
+
+    Returns
+    -------
+    List[Dict[str, str]]
+        a list of Google Classroom CourseAliases for the given course id,
+            see https://developers.google.com/classroom/reference/rest/v1/courses.aliases
+    """
+
+    if resource is None:
+        return []
```

Comment:
Line 44 is not covered by unit tests. Probably not a big deal. How much effort & value would come from adding a test?

Reply:
I think resource being Optional is outdated. That check is largely to keep mypy happy. Removed Optional and those checks, everything looks good.